content/zh/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -127,6 +127,6 @@ Endpoints API 提供了一种简单明了的方法在 Kubernetes 中跟踪网络
 * [Enabling Endpoint Slices](/docs/tasks/administer-cluster/enabling-endpoint-slices)
 * Read [Connecting Applications with Services](/docs/concepts/services-networking/connect-applications-service/)
 -->
-* [启用 Endpoint Slices](/docs/tasks/administer-cluster/enabling-endpoint-slices)
+* [启用 Endpoint Slices](/docs/tasks/administer-cluster/enabling-endpoint-slices
```

Comment:
@robscott Do you intend to add further updates this page (the YAML)?

Reply:
Woops, thanks for catching that! Originally the plan was for EndpointSlices to be enabled by default, so I'd removed this in the initial version of the PR. As it turns out, this wasn't deemed quite stable enough to be enabled by default so I had to bring this back, and I messed up adding this back in the first time.

Reply:
Not sure what happened here. Are you missing the end paren?

Reply:
That was something I missed the first time I tried to revert my initial removal of that link. I pushed a fix for that yesterday so there should be no changes to the `zh` version of this file left in this PR. Sorry for the confusion there!

Reply:
👍